src/components/ChattingScreen.js

```diff
@@ -42,6 +42,10 @@ export default function ChattingScreen() {
 	let currentSenderName = currentSender ? USER_T_NAME : USER_F_NAME;
 	let currentSenderIMG = currentSender ? USER_T_IMG : USER_F_IMG;
 
+	useEffect(() => {
+		window.scrollBy(0, window.innerHeight);
+	}, [currentMsglist]);
```

Comment:
제가 window.innerHeight를 사용해봤는데 메세지 창을 많이 생성 할때 맨 윗부분으로 
스크롤 바를 옮기고 난 후 메세지창을 입력하니 올린 대화메세지 창으로 가지 못하는 경우가 
있어서 한번 의견 남겨봅니다!!  
```suggestion
	useEffect(() => {
		window.scrollBy(0, document.body.scrollHeight);
	}, [currentMsglist]);
```

이런식으로 하면 대화메세지를 쓸 때마다 스크롤 바가 제일 밑으로 내려가 메세지를 볼 수 있습니다~!!

Reply:
우와 정말 그렇네요!!!! 좋은 조언 감사합니다 👍

Reply:
오오 꿀팁이네요